libs/ui-components/src/components/ListPage/ListPage.tsx

```
@@ -1,6 +1,15 @@
 import * as React from 'react';
 
-import { PageSection, PageSectionVariants, Title, TitleProps } from '@patternfly/react-core';
+import {
+  Flex,
+  FlexItem,
+  PageSection,
+  PageSectionVariants,
+  Split,
```

ESLint

[error] 'Split' is defined but never used.

(@typescript-eslint/no-unused-vars)